server/lib/chat/chat-service.ts

```diff
@@ -201,33 +201,22 @@ export default class ChatService {
     let channel: ChannelInfo | undefined
     let userChannelEntry: UserChannelEntry | undefined
     let message: ChatMessage
-    let error: ChatServiceError | undefined
+    let isUserBanned: boolean | undefined
```

Comment:
imo this it's better to just set this to `false`, no need for type annotations that way